infrastructure/users.tf

```diff
@@ -88,3 +97,35 @@ resource "aws_iam_user_policy" "data_refinery_user_foreman_policy" {
 }
 EOF
 }
+
+data "aws_iam_policy_document" "data-refinery-deployment" {
+  statement {
+    actions = [
+      "s3:ListObjects",
+      "s3:GetObject",
+      "s3:PutObject",
+      "s3:DeleteObject",
+      "s3:PutObjectAcl",
```

Comment:
This bucket is private but this permission makes it so the objects in it can be pushed as public, which makes the site work. Idk if we should just switch the bucket to being public though.